app/src/main/admin/components/Users/SingleUser/UserDetailsContent.tsx

```diff
@@ -81,5 +83,13 @@ export class UserDetailsContentComponent extends RemoteContentComponent<Props, u
     }
 }
 
-export const UserDetailsContent =
-    connectToStores(UserDetailsContentComponent);
\ No newline at end of file
+export const UserDetailsContentAltWrapped =
+    connectToStores(UserDetailsContentComponent);
+
+const mapStateToProps = (state: any) => {
+    return {
+        permissions: state.auth.permissions,
```

Comment:
I really like what you did on the `ListUsers` component, where `canManageUsers` was set as a prop directly. in this method. I think it'd be nice to do the same thing here with `isAdmin`